Orange/preprocess/preprocess.py

```diff
@@ -326,8 +326,7 @@ class Randomize(Preprocess):
     >>> randomizer = Randomize(Randomize.RandomizeClasses)
     >>> randomized_data = randomizer(data)
     """
-    Type = Enum("Randomize",
-                "RandomizeClasses, RandomizeAttributes, RandomizeMetas")
+    Type = 1, 2, 4
```

Comment:
Why should it be an enum? It is no longer enum, because they don't support bitwise operators.
If you insist with enums, I should probably add two new parameters to the method (for each type of randomization). Or am I missing something here?